src/haz3lcore/tiles/Secondary.re

```diff
@@ -0,0 +1,53 @@
+open Sexplib.Std;
+[@deriving (show({with_path: false}), sexp, yojson)]
+type t = {
+  id: Id.t,
+  content: secondary_content,
+}
+and secondary_content =
+  | Whitespace(string)
+  | Comment(string);
+
+let space = " ";
+let linebreak = "⏎"; //alternative: "¶"
+let comment = Re.Str.regexp("^#[^#]*#?$");
```

Comment:
I believe that it was originally a hack to get the insertion of comments to work, since otherwise I wasn't able to get it to insert the comment. I ended up forgetting about it. Right now the insertion and deletion logic should make it impossible for only a single '#' to appear, but you're right I'll work on making the code not require that.